pkgs/tools/admin/ossutil/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub, fetchpatch }:
+buildGoModule rec {
+  version = "1.7.6";
+  pname = "ossutil";
+
+  src = fetchFromGitHub {
+    owner = "aliyun";
+    repo = "ossutil";
+    rev = version;
+    sha256 = "1hkdk0hidnm7vz320i7s4z7jngx2j70acc93agii2b3r2bb91l3d";
+  };
+
+  # this patch is required to add go mods to fetch dependencies
+  patches = [
+    (
+      fetchpatch {
+        url = "https://github.com/aliyun/ossutil/commit/64067e979fb24ffb198a0c4eca718e81b63f514e.patch";
+        sha256 = "2pn0BcbNNL+iMema54LRpG/ca5kyDugLIZQ/TMhYG/8=";
+      }
+    )
```

Comment:
```suggestion
    (fetchpatch {
      url = "https://github.com/aliyun/ossutil/commit/64067e979fb24ffb198a0c4eca718e81b63f514e.patch";
      sha256 = "2pn0BcbNNL+iMema54LRpG/ca5kyDugLIZQ/TMhYG/8=";
    })
```

Reply:
nixpkgs-fmt does not hard enforce this style but you will find it in many places in nixpkgs like this.

Reply:
Sounds good, I've updated it!